app/assets/stylesheets/index.scss

```diff
@@ -61,3 +78,30 @@ a {
 .pagination {
   text-align: center;
 }
+
+
+.footer {
+  background-color: #292c2f;
+  text-align: center;
+  font-size: 1.2rem;
+  padding: 2rem; }
+  .footer p.slogan {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Merge rule `.footer p.slogan` with rule on line 83<br>Avoid qualifying class selectors with an element.